src/components/Accordion/components/Default.tsx

```diff
@@ -0,0 +1,102 @@
+import { ChevronDown } from './ChevronDown';
+import { Description } from './Description';
+import { PanelButton } from './PanelButton';
+
+interface Props {
+    title: string;
+    description?: string;
+    info?: string;
+    buttonLabel?: string;
+    expanded?: boolean;
+    children?: ReactNode;
+}
+
+const ButtonLabel = styled(Text).attrs({ as: 'p' })`
+    color: ${Colors.ACTION_BLUE_900};
+`;
+
+const PanelHeader = styled(Header)`
+    &:hover {
+        background-color: ${Colors.ACTION_BLUE_50};
+    }
+
+    &:hover ${ButtonLabel} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronDown} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+`;
+
+const CardHeader = styled(Header).attrs({ p: '3' })`
+    background-color: ${Colors.AUTHENTIC_BLUE_50};
+    border-radius: 5px 5px 0 0;
+
+    &:hover {
+        background-color: ${Colors.ACTION_BLUE_50};
+    }
+
+    &:hover ${ButtonLabel} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+
+    &:hover ${ChevronUp} {
+        color: ${Colors.ACTION_BLUE_1000};
+    }
+`;
+
+const PanelBody = styled(Box).attrs({ my: '3' })`
+    border: solid 1px ${Colors.AUTHENTIC_BLUE_200};
+    border-radius: 5px;
+`;
+
+const DefaultPanel = ({ title, description, info, buttonLabel, expanded = false, children }: Props) => {
+    const [isOpen, setIsOpen] = useState<boolean>(expanded);
+
+    return (
+        <>
+            {isOpen ? (
+                <PanelBody>
+                    <CardHeader onClick={() => setIsOpen(!isOpen)}>
+                        <Box display="flex" flexDirection="column" maxWidth="33%">
+                            <Headline as="h4" mr="3">
+                                {title}
+                            </Headline>
+                            <Description mt="1" description={description} />
+                        </Box>
+                        <PanelButton isOpen={isOpen} color={Colors.ACTION_BLUE_900}>
+                            <ButtonLabel>{buttonLabel}</ButtonLabel>
+                        </PanelButton>
+                    </CardHeader>
+                    <Box m="3">{children}</Box>
+                </PanelBody>
+            ) : (
+                <PanelHeader onClick={() => setIsOpen(!isOpen)}>
+                    <Headline as="h4" mr="3">
+                        {title}
+                    </Headline>
+                    <Box>
+                        <Description description={description} />
+                        <Text as="p" style={{ textOverflow: 'ellipsis', overflow: 'hidden', whiteSpace: 'nowrap' }}>
+                            {info}
+                        </Text>
+                    </Box>
+                    <PanelButton isOpen={isOpen} color={Colors.ACTION_BLUE_900}>
+                        <ButtonLabel>{buttonLabel}</ButtonLabel>
+                    </PanelButton>
+                </PanelHeader>
+            )}
+        </>
+    );
+};
+export default DefaultPanel;
```

Comment:
Same as for the `Compact` component

- [ ] Please use a named export for the `DefaultPanel`

Reply:
Done